src/App.tsx

```diff
@@ -89,6 +102,26 @@ const App: React.FC = () => {
                 </ProtectedRoute>
               }
             />
+            <Route
+              path="/questions/queue/technical"
+              element={
+                <ProtectedRoute>
+                  <AdminRoute>
+                    <QuestionQueueDashboard questionType={'technical'} />
```

Comment:
you don't need the {' '}

Reply:
I would also prefer an enum instead of just strings, or we can read from the url what it is instead of hardcoding?